### PR TITLE
fix(hotspot): i18n need to pass through all props downstream to components

### PR DESCRIPTION
**Pull request by scottdickerson**
Closes #

**Summary**

- This PR prereqs the HotspotModal integrated into the ImageCard so make sure to check the changes after integrating

**Change List (commits, features, bugs, etc)**

- need to pass through all i18n props (refactor them to be i18n children) and send them all the way out to the DashboardEditor component so they can be easily overridden

**Acceptance Test (how to verify the PR)**

- Verify that the DashboardEditor i18n story under Image Card with Hotspots never shows any default PII

### CODE REVIEW

**Review by JoelArmendariz — APPROVED** [review]
LGTM

File: src/components/HotspotEditorModal/HotspotEditorDataSourceTab/HotspotEditorDataSourceTab.story.jsx [hunk 1]
```diff
@@ -219,6 +220,7 @@ export const WithPresetValues = () => {
           thresholds={cardConfigState.thresholds}
           cardConfig={cardConfigState}
           dataItems={dataItems}
+          translateWithId={jest.fn()}
```
Comment (davidicus) [inline src/components/HotspotEditorModal/HotspotEditorDataSourceTab/HotspotEditorDataSourceTab.story.jsx:223]: We have a jest function in our story?

File: src/components/HotspotEditorModal/HotspotTextStyleTab/HotspotTextStyleTab.story.jsx [hunk 2]
```diff
@@ -66,6 +66,7 @@ export const Default = () => {
         onDelete={(change) => {
           action('onDelete')(change);
         }}
+        translateWithId={jest.fn()}
```
Comment (davidicus) [inline src/components/HotspotEditorModal/HotspotTextStyleTab/HotspotTextStyleTab.story.jsx:69]: Same comment about the jest function in our stories. Is this intentional?

**Review by davidicus — APPROVED** [review]
This should be good. Just want to address the jest functions in our story

**Review by davidicus — CHANGES_REQUESTED** [review]
First would like to address the jest functions in our stories

File: src/components/HotspotEditorModal/HotspotTextStyleTab/HotspotTextStyleTab.story.jsx (hunk 2 above)
Comment (scottdickerson) [reply src/components/HotspotEditorModal/HotspotTextStyleTab/HotspotTextStyleTab.story.jsx]: thanks for catching

**Review by davidicus — APPROVED** [review]

**Pull request merged**